packages/@cdktf/cli-core/src/lib/cdktf-stack.ts

```diff
@@ -385,14 +401,15 @@ export class CdktfStack {
     });
   }
 
-  public async destroy(terraformParallelism?: number) {
+  public async destroy(terraformParallelism?: number, noColor?: boolean) {
     await this.run(async () => {
       this.updateState({ type: "planning", stackName: this.stack.name });
-      const terraform = await this.initalizeTerraform({ isSpeculative: false });
```

Comment:
Not: Why did you decide against putting the noColor option into the terraform client. That would mean fewer places to remember to pass it down 🤔

Reply:
Now that you mention it, that would've been a great idea 😅. Initial thought was to follow the previous pattern, though that wasn't exactly needed since the no-color option isn't being used in various places like many of the other options.